docs/guides/contributing/tsc/meetings/2020-08-03.md

```diff
@@ -0,0 +1,75 @@
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Troy Sobotka
+  * [X] Chin-Ying Li
+
+Apologies:
+  * Mark Boorer
+  * Mei Chu
+
+# **OCIO TSC Meeting Notes**
+
+* Outstanding work to hit feature complete date:
+    - Doug: Bare-bones website running by Aug 12th (DigiPro talk), for marketing 
+      and building interest. Will be traffic following DigiPro.
+    - Michael: Any areas of docs unaccounted for?
+    - Doug: Grahame and I working on key pieces. Creating starting point, with 
+      documented high level features. Other part is landing page - Carol 
+      working on that.
+    - Carol: Looking at Hugo for landing page. Seems simple enough to set up. 
+      Hugo is better setup out of box. Easier to get started. Better support 
+      community. Static site for gh-pages. Talking design ideas with Mario, but 
+      will get something up for now.
+    - Michael: Will have PR in by end of week with documentation sections 
+      committed to.
+    - Sean: Anyone able to get docs built?
+    - Doug: Requires Python 3 so a bit problematic. Requirement for Sphinx 2, a 
+      theme dependency, which is Python 3 only. 
+    - Any other open infastructure tasks?
+    - Sean: Still working on API doc build. Current PR is for extracting 
+      docstrings from python binding, blocked by a clang depednency.
+    - Michael: Only potential issue is Windows. Clang is common on Linux, and 
+      default on macOS. What do you need to complete API work?
+    - Sean: Getting workflow down for extracting RST files from Python bindings. 
+      Only way to do it is locally generating RST files with breathe and autodoc. 
+      Need to find a way to do this at runtime - RTD limitation. Might be way 
+      for RTD build server to use Docker container with pre-built OCIO libs
+      locally available. Sphinx plugin can rebuild expanded RTS, as a fallback 
+      solution.
+    - Michael: Anything Autodesk team needs to hit date?
+    - Doug: May need help with local doc build, but will reach out. Will then 
+      try to get RST files written.
+    - Sean: There is a requirements file that pip can be pointed to to install 
+      dependencies.
+
+* OSD preparations:
+    - Michael: Any interest in talking documentaion process at OSD?
+    - Carol: Could talk angle of community involvement with docs.
+    - Michael: Can also have brief intro to docs to get interest.
+    - Doug: Good to have more participation in the BoF.
+    - Carol: Sean and I can talk and have slides about process and introducing 
+      docs. If there's no Q&A, good to introduce landing point for exploring 
+      docs. More important if there's no discussion.
+    
+* OCIO v2 WG meeting
+    - Doug: Will this Thursday work? Will send email. Number of PRs going to 
+      land soon. Want to give explanation and guidance about the work. These PRs 
```

Comment:
Please change "These PRs are remaining work to be feature complete" to "Will discuss the work remaining to be feature complete."

Reply:
Changed.